DESCRIPTION

```diff
@@ -25,6 +25,7 @@ Imports:
     lifecycle (>= 1.0.3),
     modelenv (>= 0.1.0),
     parsnip (>= 1.2.1.9000),
+    recipes (>= 1.0.10.9000),
     rlang (>= 1.1.0),
     tidyselect (>= 1.2.0),
     sparsevctrs (>= 0.1.0.9002),
```

Comment:
FYI there's a SHA of this package that doesn't have `sparsity()` exported but has version 0.1.0.9002.